packages/client/src/App.tsx

```diff
@@ -346,122 +427,120 @@ export function App(props: IAppProps) {
                                               }
                                               component={CreateNewUser}
                                             />
                                             <ProtectedRoute
                                               exact
+                                              scopes={[
+                                                SCOPES.USER_CREATE,
+                                                SCOPES.USER_CREATE_MY_JURISDICTION,
+                                                SCOPES.USER_READ_MY_OFFICE
+                                              ]}
                                               path={routes.CREATE_USER_SECTION}
                                               component={CreateNewUser}
                                             />
                                             <ProtectedRoute
                                               exact
+                                              scopes={[
+                                                SCOPES.USER_CREATE,
+                                                SCOPES.USER_CREATE_MY_JURISDICTION,
+                                                SCOPES.USER_READ_MY_OFFICE
+                                              ]}
                                               path={routes.REVIEW_USER_FORM}
                                               component={CreateNewUser}
                                             />
                                             <ProtectedRoute
                                               exact
+                                              scopes={[
+                                                SCOPES.USER_CREATE,
+                                                SCOPES.USER_CREATE_MY_JURISDICTION,
+                                                SCOPES.USER_READ_MY_OFFICE
+                                              ]}
                                               path={routes.REVIEW_USER_DETAILS}
                                               component={CreateNewUser}
                                             />
                                             <ProtectedRoute
                                               exact
-                                              // roles={[
-                                              //   SystemRoleType.RegistrationAgent,
-                                              //   SystemRoleType.LocalRegistrar,
-                                              //   SystemRoleType.LocalSystemAdmin,
-                                              //   SystemRoleType.NationalSystemAdmin,
-                                              //   SystemRoleType.PerformanceManagement,
-                                              //   SystemRoleType.NationalRegistrar
-                                              // ]}
+                                              scopes={[SCOPES.PERFORMANCE_READ]}
```

Comment:
https://github.com/opencrvs/opencrvs-core/pull/7790/files#diff-ec2dc392bcf26c2a7c1a34631db8a5f5d220596d2e986f8959acaa22494fcb78R418-R421

Reply:
Better to sync the performance & organisation ones from https://github.com/opencrvs/opencrvs-core/pull/7790/files

Reply:
Done